tools/extra/qpafilter/qpafilter.py

```diff
@@ -259,7 +301,7 @@ def create_blob_from_qpa(args):
                 sys.exit(1)
 
             for sensor in key_vals:
-                wr.write_sensor(sensor)
+                writer.write_sensor(sensor)
 
 
 def dump_blob(args):
```

Comment:
is this the human readable equivalent of the 'blob'?

Reply:
Right, the dump_blob() fn will create human-readable output, given the binary from a prior "create".